README.md

```diff
@@ -18,7 +18,7 @@ pip install maritalk
 
 # Exemplo de Uso
 
-Mostramos abaixo um exemplo simples de uso em Python.
+Mostramos abaixo um exemplo simples de uso em Python. Na [pasta exemplos](./examples/api/) existem mais códigos mostrando como chamar a API.
```

Comment:
```suggestion
Mostramos abaixo um exemplo simples de uso em Python. Na [pasta exemplos](./examples/api/) existem mais códigos mostrando como chamar a API usando JavaScript/Node.js.
```

Reply:
If you prefer to be more generic in the description, feel free to ignore my suggestion...

Reply:
I think we should maintain it more generic since we may add more examples in the future.